src/main.rs

```diff
@@ -194,12 +194,9 @@ mod app {
         let pwm = c.local.pwm;
         for ch in OutputChannelIdx::into_enum_iter() {
             let s = settings.output_channel[ch as usize];
-
-            // set current_limit_positive to absolute 5% higher than iir y_max and clamp output to valid range.
+            // set current limits to 5% higher/lower than iir y_max/y_min and clamp output to valid range.
             let current_limit_positive = (s.iir.y_max as f32 + 0.05 * 3.0).clamp(0.0, 3.0);
```

Comment:
3.0 isn't correct here. Please derive the actual equivalent physical PWM limit.

Reply:
I see, thanks for the detailed comment.

Reply:
It's 6.6 A.